ctk/common/src/main/java/org/codice/compliance/utils/sign/SimpleSign.java

```diff
@@ -56,21 +58,20 @@
 import org.opensaml.xmlsec.signature.support.SignatureConstants;
 import org.opensaml.xmlsec.signature.support.SignatureValidator;
 import org.opensaml.xmlsec.signature.support.provider.ApacheSantuarioSignatureValidationProviderImpl;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public class SimpleSign {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(SimpleSign.class);
-
   private final SystemCrypto crypto;
 
-  private static final Map<String, String> URI_ALG_MAP = new HashMap<>();
-
-  static {
-    URI_ALG_MAP.put("http://www.w3.org/2000/09/xmldsig#dsa-sha1", "SHA1withDSA");
-    URI_ALG_MAP.put("http://www.w3.org/2000/09/xmldsig#rsa-sha1", "SHA1withRSA");
-  }
+  private static final ImmutableMap<String, String> URI_ALG_MAP =
```

Comment:
✏️ Should rename map or add a comment so the source of these mapped values isn't a mystery